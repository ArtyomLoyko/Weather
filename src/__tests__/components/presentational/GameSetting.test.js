import React from 'react';
import { shallow } from 'enzyme';
import GameSetting from './../../../components/presentational/GameSetting';

describe('GameSetting', () => {
    const gameSetting = shallow(<GameSetting />);

    it('renders properly', () => {
        expect(gameSetting).toMatchSnapshot();
    });
});
