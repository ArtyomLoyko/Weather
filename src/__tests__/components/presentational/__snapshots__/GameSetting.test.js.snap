// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GameSetting renders properly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <GameSetting />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h2>
          Game Setting
</h2>,
        <figure
          className="skirt-cards"
>
          <figcaption>
                    Select Scirt:
          </figcaption>
          <div>
                    <Connect(ScirtsListContainer) />
          </div>
</figure>,
      ],
      "className": "game-setting",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Game Setting",
        },
        "ref": null,
        "rendered": "Game Setting",
        "type": "h2",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <figcaption>
              Select Scirt:
</figcaption>,
            <div>
              <Connect(ScirtsListContainer) />
</div>,
          ],
          "className": "skirt-cards",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Select Scirt:",
            },
            "ref": null,
            "rendered": "Select Scirt:",
            "type": "figcaption",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <Connect(ScirtsListContainer) />,
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {},
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            "type": "div",
          },
        ],
        "type": "figure",
      },
    ],
    "type": "section",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h2>
            Game Setting
</h2>,
          <figure
            className="skirt-cards"
>
            <figcaption>
                        Select Scirt:
            </figcaption>
            <div>
                        <Connect(ScirtsListContainer) />
            </div>
</figure>,
        ],
        "className": "game-setting",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Game Setting",
          },
          "ref": null,
          "rendered": "Game Setting",
          "type": "h2",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <figcaption>
                Select Scirt:
</figcaption>,
              <div>
                <Connect(ScirtsListContainer) />
</div>,
            ],
            "className": "skirt-cards",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Select Scirt:",
              },
              "ref": null,
              "rendered": "Select Scirt:",
              "type": "figcaption",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <Connect(ScirtsListContainer) />,
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {},
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
              "type": "div",
            },
          ],
          "type": "figure",
        },
      ],
      "type": "section",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    "attachTo": undefined,
    "hydrateIn": undefined,
  },
}
`;
